src/sentry/db/deletion.py

```diff
@@ -15,39 +31,81 @@ def __init__(self, model, project_id=None, dtfield=None, days=None):
         self.days = int(days) if days is not None else None
         self.using = router.db_for_write(model)
 
-    def execute_postgres(self, chunk_size=10000):
-        quote_name = connections[self.using].ops.quote_name
+    def execute_postgres(self, chunk_size=10000, chunk_delta=None):
```

Comment:
`delta` is a pretty ambiguous term here, it seems like `duration` might be more appropriate.